lib/ansible/modules/cloud/misc/virt_net.py

```diff
@@ -556,7 +560,10 @@ def core(module):
             return VIRT_SUCCESS, res
 
         elif hasattr(v, command):
-            res = getattr(v, command)()
+            if command == 'facts' and name:
+                res = getattr(v, command)(name)
```

Comment:
you don't need the `getattr()` here, `v.facts(name)` is enough and would improve the readability.

Reply:
done